base_app/base.py

```diff
@@ -36,5 +38,33 @@ def index():
 
     return result
 
+@app.route('/api/v1/photos', methods=['GET'])
+def get_photos():
+    YELP_API_KEY = os.getenv('YELP_API_KEY')
+    headers = {'authorization': 'Bearer ' + YELP_API_KEY}
+    lat = request.args['latitude']
+    long = request.args['longitude']
+    cuisine = request.args['categories']
+    params = {'latitude': f'{lat}', 'longitude': f'{long}', 'term': f'{cuisine}'}
+    url = 'https://api.yelp.com/v3/businesses/search'
+
+    response = requests.get(url, params=params, headers=headers)
+    json_data = json.dumps(response.json())
+    restaurants = Restaurant.from_json(json_data)
+    restaurant = random.choice(restaurants)
+    schema = RestaurantSchema()
+    result = schema.dump(restaurant)
+
+    restaurant_id = result['data']['id']
+    restaurant_url = f'https://api.yelp.com/v3/businesses/{restaurant_id}'
+
+    response = requests.get(restaurant_url, headers=headers)
+    json_data = json.dumps(response.json())
+    photos = Photo.from_json(json_data)
```

Comment:
Great catch noticing that all the info we need is in the /business/id endpoint from yelp. I think on our next refactor we can address this pretty easily (we can probably get an id from the /businesses/search endpoint and pass it to this one, etc.).